### DISCUSSION TITLE
Standings May 10, 2022 - before the deadline!

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas22_competition#56] [Announcements]
| Rank | Team   | Points |
|------|---------|------------|
| 1.   | Team40 |  85,5       |
| 2.   | Team02  |  84,5        |
| 3.   | Team38  | 78,5        |
| 4.   | Team04  |  75,5        |
| 5.   | Team50 |  71,5       |
| 6.   | Team36  |  67*       |
| 7.   | Team52  |  63,5        |
| 8.   | Team53  |  59,5        |
| 9.   | Team13 |  59       |
| 10.   | Team10  |  55,5        |
| 11.   | Team49  |  N/A        |
| 12.   | Team46  |  N/A        |
| 13.   | Team39  |  N/A        |
| 14.   | Team31  |  N/A        |
| 15.   | Team22  |  N/A        |

**Note:** The standings reflect the best runs at the moment.

**Comment by afaroo01** [comment]
Hi, are these standing based on new arena as well or on the arena 3 only?

**Reply by fpetric** [reply]
The standings are based on the evaluation arena. Arena 3 results are provided for teams to see how others perform in "known" environment.

**Reply by afaroo01** [reply]
Thanks, got it. But regards to arena 4 & 5 they are more challenging than 1,2 or 3 we had been working on previously. Our solution doesn't seem to scale for new ones added recently. Will that be an issue in evaluation?

**Reply by fpetric** [reply]
Evaluation arena is not as complex as 4 and 5, those are provided for debugging. Evaluation arena is closer in complexity to arena 3, a bit more challenging.

**Reply by fpetric** [reply]
See discussion  here for a bit more details:
https://github.com/larics/icuas22_competition/discussions/57

**Comment by Maik93** [comment]
Does the final code will be executed on several evaluation scenarios, with e.g. random spawned obstacle and/or marker positions? This would improve the entire evaluation procedure in terms of solution's generalisation and overall performance among different test cases.

**Reply by fpetric** [reply]
Hi, 

we will definitely run the evaluation with different marker positions and different takeoff positions. We may use different layouts, but we currently use a layout that looks rather different from different starting positions so we may skip on changing obstacle layouts. But we have few arenas ready if needed.
